tests/integration/aic.rs

```diff
@@ -0,0 +1,242 @@
+use cjms::{
+    controllers::aic::AICResponse,
+    models::aic::{AICModel, AIC},
+};
+use serde_json::json;
+use time::OffsetDateTime;
+use uuid::{Uuid, Version};
+
+use crate::utils::{
+    random_ascii_string, send_get_request, send_post_request, send_put_request, spawn_app, TestApp,
+};
+
+#[tokio::test]
+async fn test_aic_get_is_not_allowed() {
+    let app = spawn_app().await;
+    let test_cases = vec!["/aic", "/aic/123"];
```

Comment:
What is the purpose of using a vector here? Can this not just be a normal array like you have in `aic_create_with_bad_data`?

Reply:
agree.

Reply:
c65191e